spec/support/api/schemas/artists.json

```diff
@@ -25,15 +25,27 @@
           "spotify_uri": {
             "id": "http://spotify-apprentice-app.com/artists/0/spotify_uri",
             "type": "string"
+          },
+          "song_ids": {
+            "id": "http://spotify-apprentice-app.com/artists/0/song_ids",
+            "type": "array",
+            "items": {
```

Comment:
you can also specify `"minItems": 0` here to make sure that it allows empty arrays for song_ids